pkg/storage/unified/sql/data/resource_history_get.sql

```diff
@@ -16,6 +16,6 @@ WHERE 1 = 1
   AND {{ .Ident "action" }} = 3
   {{ end }}
   {{ if (gt .StartRV 0) }}
-  AND {{ .Ident "resource_version" }} > {{ .Arg .StartRV }}
+  AND {{ .Ident "resource_version" }} <= {{ .Arg .StartRV }}
```

Comment:
I have a feeling this should be `<` and not `<=`. Could we add an e2e test for Listing history (in `test/integration_test.go`), it would make it easier to reason about it ?

Reply:
It was originally not returning the last element twice due to the token creation inside of the iter.Next(), but looking into it more, it caused an issue in normal list. I change the token creation back to where it was, changed this to `<`, created an e2e test, and tested out normal list. We should be good to go now. Really good catch, thank you